app/graphql/types/query_type.rb

```diff
@@ -15,5 +23,14 @@ def get_users
     def get_user(id:)
       User.find(id)
     end
+
+    # Event queries
+    def get_events
```

Comment:
Since you added Events to User type we probably don't need a query called getEvents. I can't think of a use case where we'd want to fetch all events for all users, but lemme know if there's something I'm not thinking of!